test/suite/test_s3_store01.py

```diff
@@ -47,9 +50,9 @@ def test_local_basic(self):
         s3_store = self.get_s3_storage_source()
 
         fs = s3_store.ss_customize_file_system(session, "wt-bucket", "Secret", None)
+        objlist = fs.fs_directory_list(session, self.bucket_name, '')
```

Comment:
`objlist` does not seem to be used

Reply:
I have used `objlist` as a variable just to hold the result from `fs_directory_list` after making sure that `fs_directory_list` can run without erroring. Once we have a shared bucket for testing, or we are able to insert/flush files into the bucket, we would be able to add asserts on the list of objects returned from `fs_directory_list`, to check `objlist` contains what we expect.

Reply:
Can we use the syntax `_ = fun()` for now? Or any other method for unused variables.